views/snippets/nav.html

```diff
@@ -5,8 +5,14 @@
             <a class="nav-item nav-link active" href="/">Home</span></a>
             <a class="nav-item nav-link" href="/job-roles">Jobs</span></a>
             <a class="nav-item nav-link" href="/">About Us</span></a>
-            <a class="nav-item nav-link" href="/loginForm">Login</span></a>
-            <a class="nav-item nav-link" href="/logoutForm">Logout</span></a>
+            <div class="right"></div>
+            {% if not token.length %}
```

Comment:
change to if else

Reply:
Just changed that to a If and Else statement